configs/ubuntu16_04_clang/9.0.0/bazel_0.23.1/cc/BUILD

```diff
@@ -14,12 +14,11 @@
 
 # This becomes the BUILD file for @local_config_cc// under non-FreeBSD unixes.
 
+package(default_visibility = ["//visibility:public"])
```

Comment:
This is weird. Doesn't buildifier enforce load to be at top of BUILDs?

Reply:
buildifier 0.22.0 has a bug I think. See my comment https://github.com/bazelbuild/buildtools/issues/575#issuecomment-471038826